os/hal/ports/AT32/AT32F415/at32_isr.h

```diff
@@ -104,11 +104,6 @@
 #define AT32_DMA2_CH6_NUMBER         AT32_DMA2_CH6_7_NUMBER
 #define AT32_DMA2_CH7_NUMBER         AT32_DMA2_CH6_7_NUMBER
 
-#define AT32_DMA2_CH4_CMASK          0x00000C00U
-#define AT32_DMA2_CH5_CMASK          0x00000C00U
-#define AT32_DMA2_CH6_CMASK          0x00003000U
-#define AT32_DMA2_CH7_CMASK          0x00003000U
-
```

Comment:
Maybe it was used for nvic management.

Reply:
At `R113-R127` in `at32_dma.c` files it already have the CMASK callout for DMA2 Channel 4/5/6/7 already, so those are seemly just not really needed anymore.

Reply:
But DMA2 channel 4/5 and 6/7 both use the same irq vector number, the default settings in at32_dma.c are independent.

Reply:
Oh, hmm. So maybe ChibiOS forgot about that? Since on STM32F1 code, there wasn't a CMASK call for DMA2 Channel 4/5 since both using the same IRQ vector number. But instead they are using independent on `stm32_dma.c`.

Reply:
Perhaps it exists in stm32_registry.h, such as stm32f1, line 748.

Reply:
Oh yeah, they do. For some reason it only like this:
<details>
  <summary>Click to expand!</summary>

| MCU         | CMASK |
| ----------- | ----- |
| STM32F101xC | Yes   |
| STM32F101xD | Yes   |
| STM32F101xE | Yes   |
| STM32F101xF | No    |
| STM32F101xG | No    |
| STM32F103xC | Yes   |
| STM32F103xD | Yes   |
| STM32F103xE | Yes   |
| STM32F103xF | No    |
| STM32F103xG | No    |
</details>
After some checking history commit, it seem they forget and no longer update those since Oct 22, 2015. Alright, I'll re-added those CMASK back and merged this commit. Thanks for reviewing this.